flink-runtime-web/web-dashboard/src/app/pages/job-manager/log-detail/job-manager-log-detail.component.less

```diff
@@ -16,11 +16,40 @@
  * limitations under the License.
  */
 
+@import "theme";
+
+:host {
+  display: block;
+  height: 100%;
+  &.full-screen {
+    position: fixed;
+    top: 0;
+    bottom: 0;
+    right: 0;
+    left: 0;
+    background: @component-background;
+    z-index: 99;
+    flink-monaco-editor {
+      height: calc(~"100vh - 65px");
+    }
+  }
+}
+
 flink-monaco-editor {
-  height: calc(~"100vh - 160px");
+  height: calc(~"100vh - 220px");
 }
 
-:host {
+.breadcrumb {
+  background: @component-background;
+  border-bottom: 1px solid @border-color-split;
+  margin-bottom: 16px;
+  padding: 12px 24px;
   position: relative;
-  display: block;
+}
+
+flink-refresh-download {
+  position: absolute;
+  right: 12px;
+  top: 0;
+  line-height: 47px;
```

Comment:
We seem to have done some duplicate work with `Task Managers` log list, can we make these codes reusable?

Reply:
these two components look very similar, but the style is not the same(some border and full-screen style need to adjust according to different layout)
and they may have more different features in the future, like https://github.com/apache/flink/pull/10228, I think it would better not to create a share component between them